src/components/Dashboard/StaffNotation.vue

```diff
@@ -89,6 +97,13 @@
                 </v-list-item-title>
               </v-list-item>
             </template>
+            <template v-if="isFirm || isCoop || isBenBcCccUlc">
+              <v-list-item v-if="!isHistorical && !isAdminFreeze" @click="showAdministerFreezeDialog()">
```

Comment:
So this item is not displayed if the business is not frozen? Then why does the dialog support both freezing and unfreezing?

Reply:
Also, after you rebase, please update your new code to match the other templates/v-list-items. One notable change is that the v-if on line 101 should be moved to line 100 (so we don't get an empty template if the condition is false).